scale-tests/run.sh

```diff
@@ -0,0 +1,434 @@
+      --num-producers-per-kafka "${FINITE_NUM_PRODUCERS_PER_KAFKA}" \
+      --num-consumers-per-producer "${FINITE_NUM_CONSUMERS_PER_PRODUCER}" \
+      --producer-number-of-words "${FINITE_PRODUCER_NUMBER_OF_WORDS}" \
+      --producer-words-per-second "${FINITE_PRODUCER_WORDS_PER_SECOND}" \
+      --producer-spark-cores-max "${FINITE_PRODUCER_SPARK_CORES_MAX}" \
+      --producer-spark-executor-cores "${FINITE_PRODUCER_SPARK_EXECUTOR_CORES}" \
+      --consumer-write-to-cassandra \
+      --consumer-batch-size-seconds "${FINITE_CONSUMER_BATCH_SIZE_SECONDS}" \
+      --consumer-spark-cores-max "${FINITE_CONSUMER_SPARK_CORES_MAX}" \
+      --consumer-spark-executor-cores "${FINITE_CONSUMER_SPARK_EXECUTOR_CORES}"
+
+  log 'Uploading finite jobs submissions file'
+  container_exec \
+    aws s3 cp --acl public-read \
+      "${FINITE_SUBMISSIONS_OUTPUT_FILE}" \
+      "s3://${TEST_S3_BUCKET}/${TEST_S3_FOLDER}/"
+else
+  log 'Skipping running of finite streaming jobs'
+fi
+
+if [ "${SHOULD_RUN_INFINITE_STREAMING_JOBS}" = true ]; then
+  log 'Running infinite jobs. Consumers do not write to Cassandra'
+  container_exec \
+    ./scale-tests/kafka_cassandra_streaming_test.py \
+      "${DISPATCHERS_JSON_OUTPUT_FILE}" \
+      "${INFRASTRUCTURE_OUTPUT_FILE}" \
+      "${INFINITE_SUBMISSIONS_OUTPUT_FILE}" \
+      --jar "${TEST_ASSEMBLY_JAR_URL}" \
+      --num-producers-per-kafka "${INFINITE_NUM_PRODUCERS_PER_KAFKA}" \
+      --num-consumers-per-producer "${INFINITE_NUM_CONSUMERS_PER_PRODUCER}" \
+      --producer-number-of-words "${INFINITE_PRODUCER_NUMBER_OF_WORDS}" \
+      --producer-words-per-second "${INFINITE_PRODUCER_WORDS_PER_SECOND}" \
+      --producer-spark-cores-max "${INFINITE_PRODUCER_SPARK_CORES_MAX}" \
+      --producer-spark-executor-cores "${INFINITE_PRODUCER_SPARK_EXECUTOR_CORES}" \
+      --consumer-batch-size-seconds "${INFINITE_CONSUMER_BATCH_SIZE_SECONDS}" \
+      --consumer-spark-cores-max "${INFINITE_CONSUMER_SPARK_CORES_MAX}" \
+      --consumer-spark-executor-cores "${INFINITE_CONSUMER_SPARK_EXECUTOR_CORES}"
+
+  log 'Uploading infinite jobs submissions file'
+  container_exec \
+    aws s3 cp --acl public-read \
+      "${INFINITE_SUBMISSIONS_OUTPUT_FILE}" \
+      "s3://${TEST_S3_BUCKET}/${TEST_S3_FOLDER}/"
+else
+  log 'Skipping running of infinite streaming jobs'
+fi
+
+if [ "${SHOULD_RUN_BATCH_JOBS}" = true ]; then
+  log 'Running batch jobs'
+  container_exec \
+    ./scale-tests/deploy-batch-marathon-app.py \
+      --app-id "${BATCH_APP_ID}" \
+      --dcos-username "${DCOS_USERNAME}" \
+      --dcos-password "${DCOS_PASSWORD}" \
+      --security "${SECURITY}" \
+      --input-file-uri "${DISPATCHERS_JSON_OUTPUT_FILE_URL}" \
+      --script-cpus "${BATCH_SCRIPT_CPUS}" \
+      --script-mem "${BATCH_SCRIPT_MEM}" \
+      --script-args "\"\
+        ${DISPATCHERS_JSON_OUTPUT_FILE} \
+        --submits-per-min ${BATCH_SUBMITS_PER_MIN} \
+      \"" \
+      --spark-build-branch "${BATCH_SPARK_BUILD_BRANCH}"
```

Comment:
Nit: Let's keep the `--script-args` as the last argument.

Reply:
https://github.com/mesosphere/spark-build/pull/348/commits/aa0310b9457aa0a421e93eb1405042c3d981353f